tools/generator/src/Generator/DisambiguateTargets.swift

```diff
@@ -481,7 +481,7 @@ extension ConsolidatedTarget {
     /// The normalized label is used during target disambiguation. It allows the
     /// logic to differentiate targets where the names only differ by case.
     var normalizedLabel: String {
-        return label.lowercased()
+        return "\(label)".lowercased()
     }
 }
```

Comment:
Is this `normalizedLabel` a one-off function? I considered adding a normalized property to `BazelLabel`.

Reply:
Yeah, it's only needed here.